src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -136,9 +142,16 @@
         <!-- and create field for none translated language -->
         <xsl:for-each select="$metadataOtherLanguages/lang">
           <xsl:variable name="currentLanguageId" select="@id"/>
+          <xsl:variable name="currentLanguageId_none_iso">
+            <xsl:choose>
+              <xsl:when test="$currentLanguageId = 'fre'">fra</xsl:when>
+              <xsl:otherwise><xsl:value-of select="$currentLanguageId"/></xsl:otherwise>
+            </xsl:choose>
+          </xsl:variable>
+
           <xsl:if test="count($theElement/
                 gmd:PT_FreeText/gmd:textGroup/
-                gmd:LocalisedCharacterString[@locale = concat('#',$currentLanguageId)]) = 0">
+                gmd:LocalisedCharacterString[@locale = concat('#',$currentLanguageId_none_iso)]) = 0">
```

Comment:
Odd to use none_ISO for a variable name. Why not rename the other variable to ISO?

Reply:
I thought about it but the other variable will the locale attribute and it's on the xpath. So would this be better?

<variable name = "local_ISO" select ="$theElement/gmd:PT_FreeText/gmd:textGroup/gmd:LocalisedCharacterString/@locale">?

Then use this to compare?

Reply:
I have rename the variable to currentLanguageId_iso639_2t to indicate it is iso639-2t format.